src/Core/src/Hosting/Internal/FallbackLogging.Debug.cs

```diff
@@ -0,0 +1,19 @@
+#nullable enable
+#define DEBUG
```

Comment:
The `Debug.WriteLine()`will be removed in release builds, so make this file always in a debug build. The compiler will never know the difference!